runelite-client/src/main/java/net/runelite/client/plugins/hiscore/HiscorePanel.java

```diff
@@ -306,171 +226,103 @@ public void mouseClicked(MouseEvent e)
 					iconImage = ImageIO.read(HiscorePanel.class.getResourceAsStream(
 						endpoint.name().toLowerCase() + ".png"));
 				}
-				JToggleButton button = new JToggleButton();
-				button.setIcon(new ImageIcon(iconImage));
-				button.setPreferredSize(new Dimension(24, 24));
-				button.setBackground(Color.WHITE);
-				button.setFocusPainted(false);
-				button.setActionCommand(endpoint.name());
-				button.setToolTipText(endpoint.getName() + " Hiscores");
-				button.addActionListener((e -> executor.execute(this::lookup)));
-				button.addMouseListener(new MouseAdapter()
+				JPanel panel = new JPanel();
+				JLabel label = new JLabel();
+
+				label.setIcon(new ImageIcon(iconImage));
+
+				panel.add(label);
+				panel.setBackground(ColorScheme.DARKER_GRAY_COLOR);
+				panel.setToolTipText(endpoint.getName() + " Hiscores");
+				panel.addMouseListener(new MouseAdapter()
 				{
 					@Override
-					public void mouseReleased(MouseEvent e)
+					public void mouseClicked(MouseEvent e)
 					{
+						if (loading)
+						{
+							return;
+						}
+
+						executor.execute(new Runnable()
```

Comment:
executor.execute(HiscorePanel.this::lookup)

Reply:
ah! I'm not familiar with that syntax, thanks.